cluster_test.go

```diff
@@ -773,6 +773,43 @@ func TestClusterPins(t *testing.T) {
 	}
 }
 
+func TestClusterPinsWithExpiration(t *testing.T) {
```

Comment:
This test is more valuable if run with several clusters (as part of ipfscluster_test.go). It will cover more code.